main.cpp

```diff
@@ -107,10 +107,21 @@ struct CarWash
     you should be able to deduce the return type of those functions based on their usage in Person::run()
     You'll need to insert the Person struct from the video in the space below.
  */
-
-
-
-
+   struct Limb
+   {
+       void stepForward();
+       float stepSize();
+   };
+   struct Person
+   {
+       int age;
+       int height;
+       float hairLength;
+       float GPA;
+       unsigned int SATScore;
+       Limb leftFoot;
+       Limb rightFoot;
+   };
```

Comment:
```
    don't forget to define and implement the member functions 'stepForward()' and 'stepSize()'
```
You also need to copy over the implementation of `Person::run()` from the video.
- [ ] revise